ckan/public-midnight-blue/base/javascript/client.js

```diff
@@ -48,14 +48,17 @@
         success: fn,
         error: error
       };
-      if (type == 'POST') {
-        options.type = 'POST';
-        options.data = JSON.stringify(data);
-        var csrf_field = $('meta[name=csrf_field_name]').attr('content');
-        var csrf_token = $('meta[name='+ csrf_field +']').attr('content');
-        options.headers = {
-          'X-CSRFToken': csrf_token
-        }
+      if (type === 'POST') {
```

Comment:
I need a second set of eyes to review if this is valid promise and not skip the first try? or if it needs to be rejigged so its not ignored since its gone async style.

Reply:
Also missed: https://github.com/ckan/ckan/blob/master/ckan/public/base/javascript/htmx-ckan.js